test/apps/iodemo/io_demo.cc

```diff
@@ -605,7 +674,10 @@ class DemoClient : public P2pDemoCommon {
              std::string> _status_str;
     double                _start_time;
     unsigned              _retry;
+public:    
+    static MemoryPool<IoReadResponseCallback> rCallbackPool;
```

Comment:
1. should not be static
2. rename to _callback_pool

Reply:
Fixed. Thanks!